flag_test.go

```diff
@@ -30,12 +30,42 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type flagTestCase struct {
+	args      []string
+	wantLevel zapcore.Level
+	wantErr   bool
+}
+
+func (tc flagTestCase) RunImplicitSet(t testing.TB) {
```

Comment:
True.

I keep wishing that the Go community had some convention for intra-package visibility hints, but I don't think that's happening anytime soon.